com.unity.render-pipelines.core/Runtime/Debugging/DebugFrameTiming.cs

```diff
@@ -0,0 +1,199 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+//#define RTPROFILER_DEBUG
+
+namespace UnityEngine.Rendering
+{
+    public class DebugFrameTiming
+    {
+        const string k_FpsFormatString = "{0:F1}";
+        const string k_MsFormatString = "{0:F2}ms";
+        const float k_RefreshRate = 1f / 5f;
+
+        internal FrameTimeSampleHistory m_FrameHistory = new();
+        internal BottleneckHistory m_BottleneckHistory = new();
+
+        /// <summary>
+        /// Size of the Bottleneck History Window in number of samples.
+        /// </summary>
+        public int BottleneckHistorySize { get; set; } = 60;
+
+        /// <summary>
+        /// Size of the Sample History Window in number of samples.
+        /// </summary>
+        public int SampleHistorySize { get; set; } = 30;
```

Comment:
Same here, properties start with lowercase
```suggestion
        public int sampleHistorySize { get; set; } = 30;
```

Reply:
Thanks, fixed.